sdk/python/requirements.txt

```diff
@@ -2,7 +2,7 @@
 # This file is autogenerated by pip-compile with python 3.9
 # To update, run:
 #
-#    pip-compile requirements.in
+#    pip-compile --no-emit-index-url requirements.in
```

Comment:
is this needed?

Reply:
It seems advisable... if you don't include that and you have custom (private) PyPi indexes configured, they end up appearing in the generated code. Happy to remove it if you prefer.

Reply:
Got it, thanks for the explanation.